src/services/tx/safeUpdateParams.ts

```diff
@@ -32,6 +34,10 @@ const getChangeFallbackHandlerCallData = async (
 export const createUpdateSafeTxs = async (safe: SafeInfo, chain: ChainInfo): Promise<MetaTransactionData[]> => {
   assertValidSafeVersion(safe.version)
 
+  if (semverSatisfies(safe.version, '>=1.3.0')) {
```

Comment:
Are we sure this will also work with future safe version e.g. going from 1.4.1 to 1.5.0? Otherwise I would limit it to only target 1.3.0 versions instead of >=1.3.0